src/connector/src/source/mqtt/source/reader.rs

```diff
@@ -64,8 +64,8 @@ impl SplitReader for MqttSplitReader {
             .subscribe_many(
                 splits
                     .iter()
-                    .cloned()
-                    .map(|split| Filter::new(split.topic, qos)),
+                    .flat_map(|split| split.topic.split(","))
```

Comment:
Is it possible to split the topic on the CN side without having to split it? Because it is a static topic string, it is directly divided on the meta side, allowing multiple cns to consume different topics and improve throughput

Reply:
Possible yes, though I intentionally chose not to: (from the PR description)
> I intentionally chose to keep 1 split, even when using multiple MQTT topics to prevent message duplication in the case of overlapping topics. e.g. subscribing to /a/# and /a/b/# should only result in 1 message on topic a/b/c. Splitting multiple topics could result in message duplication, if one client subscribed to /a/# and the other subscribed to /a/b/#